src/tek-poe.h

```diff
@@ -15,6 +15,9 @@
 
 struct port_state {
 	const char *status;
+	float mvolt;
+	float mampere;
+	float tempc;
```

Comment:
`temp_celsius` if you intend to keep the temperature.

Reply:
that's what it is according to the protocol, yes

Reply:
Certain parts of the world might like a redundant `temp_fahrenheit`  too.

Reply:
Yeah, but that would be more something for the rendering stage, I think.  Could report double, or use config to decide.